pymc3/tests/test_distributions.py

```diff
@@ -909,17 +909,12 @@ def test_multinomial_mode(self, p, n):
         [[.25, .25, .25, .25], (1, 4), 3],
         # 3: expect to fail
         # [[.25, .25, .25, .25], (10, 4)],
-        [[.25, .25, .25, .25], (10, 1, 4), 5],
```

Comment:
So I take it these cases are now not supported?

Reply:
I was in a dark place, but oddly these all "just pass" now.